cl/users/templates/profile/webhooks.html

```diff
@@ -1,68 +1,96 @@
 {% extends "profile/nav.html" %}
+{% load static %}
 {% load waffle_tags %}
 
 {% block title %} Developer Tools &ndash; CourtListener.com{% endblock %}
 
 {% block nav-api %}active{% endblock %}
+{% block footer-scripts %}
+  {% if DEBUG %}
+    <script src="{% static "js/htmx.js" %}"></script>
+    <link href="{% static "css/bootstrap4-toggle.css" %}" rel="stylesheet">
```

Comment:
Did you end up using this?

Reply:
no, we don't use the toggle here, removed.